manager/controllers/default/system/info.class.php

```diff
@@ -42,9 +42,10 @@ public function process(array $scriptProperties = array()) {
         if ($dbtype_mysql && !empty($m['pdo_mysql'])) $pi = array_merge($pi,array('pdo_mysql' => $m['pdo_mysql']));
         if ($dbtype_sqlsrv && !empty($m['pdo_sqlsrv'])) $pi = array_merge($pi,array('pdo_sqlsrv' => $m['pdo_sqlsrv']));
         if (!empty($m['zip'])) $pi = array_merge($pi,array('zip' => $m['zip']));
+        $this->modx->getService('PHPMailer', 'mail.modPHPMailer');
```

Comment:
To keep it consistent with the other places in the core where we use the mailer:
```suggestion
        $this->modx->getService('mail', 'mail.modPHPMailer');
```